Analyze folder updates

tmhastings opened the pull request:
Task List for this PR

- [x] spelling, grammar, and syntax are correct
- [x] formatting is correct
- [x] information is up-to-date
- [x] links verified
- [x] images updated as needed
- [ ] @MikePlante1 or @tmhastings verify formatting with test docs

tmhastings committed:
Update EvaluateBasal.md

tmhastings committed:
Update EvaluateCR.md

tmhastings committed:
Update EvaluateBasal.md

tmhastings committed:
Update EvaluateBasal.md

tmhastings committed:
Update EvaluateCR.md

tmhastings committed:
Merge pull request #10 from nightscout/dev

Sync Branch

tmhastings committed:
Merge pull request #15 from nightscout/dev

Sync with dev

tmhastings committed:
Update EvaluateBasal.md

Syntax Edits

tmhastings committed:
Update EvaluateCR.md

Syntax Edits

tmhastings committed:
Update EvaluateBasal.md

Format Admonition for consistency

tmhastings committed:
Update EvaluateCR.md

Format Admonition for consistency
Small Syntax correction

tmhastings committed:
Update EvaluateBasal.md

Update formatting

tmhastings committed:
Update EvaluateBasal.md

Minor grammar edit

tmhastings committed:
Update EvaluateCR.md

Rephrase for clarity

tmhastings committed:
Update EvaluateBasal.md

tmhastings committed:
Update EvaluateBasal.md

Rewrite evaluate basal instructions.

tmhastings committed:
Update EvaluateCR.md

Reorganize and update information for clarity and accuracy.

MikePlante1 reviewed (changes requested):
Not sure how well markdown and html play together, so I'd either use `<b>` and`<I>`, or `**` and `*`, but not both.

tips/warnings look right

Images took me a lot of trial and error to get them to size. Here's an example of what seemed to work, but it'll be easier to get the right size once it's public and viewable on RTD: https://github.com/Artificial-Pancreas/iAPSdocs/pull/141/files

MikePlante1 committed:
Update EvaluateBasal and EvaluateCR

Changed `*` to `<i>`

Changed image formatting and dropped image into img subfolder

Minor rewordings

tmhastings committed:
Update EvaluateBasal.md

Rephrase for clarity.

tmhastings committed:
Update EvaluateCR.md

rephrase for clarity

Commit:
Update EvaluateBasal.md

Updated some language

Commit:
Update EvaluateCR.md

tmhastings committed:
Update EvaluateBasal.md

Revert all caps of “warning”

tmhastings committed:
Update EvaluateCR.md

Revert all caps “warning”

MikePlante1 committed:
rename to Trio
